monkey/infection_monkey/monkey.py

```diff
@@ -79,12 +79,24 @@ def initialize(self):
         self._network = NetworkScanner()
         self._dropper_path = sys.argv[0]
 
+        server_num=0     
+        secondary_servers= len(self._args) 
```

Comment:
not all command arguments are server ip's. Look above: there is `-p` for parrent argument, `-t` for tunnel argument and so on... I'm pretty sure this doesn't give you a number of servers passed with `-s` flag.

Reply:
Noticed that..  Working on that ..
Thanks for your reviews .. :-)

Reply:
This is not solve. Why is it called `secondary_servers` if it's argument list?

Reply:
Corrected